cg/store/api/find_business_data.py

```diff
@@ -316,6 +313,16 @@ def get_sample_by_customer_and_name(
             name=sample_name,
         ).first()
 
+    def get_number_of_reads_for_sample_from_metrics(self, sample_internal_id: str) -> int:
+        """Get number of reads for sample from sample lane sequencing metrics."""
+        total_reads = (
+            self._get_query(table=SampleLaneSequencingMetrics)
+            .with_entities(func.sum(SampleLaneSequencingMetrics.sample_total_reads_in_lane))
+            .filter(SampleLaneSequencingMetrics.sample_internal_id == sample_internal_id)
```

Comment:
Should this filter not be defined in the filters module to keep it DRY and unit tested?